src/prommis/nanofiltration/diafiltration.py

```diff
@@ -134,12 +125,71 @@ def build_model():
     m.fs.stage1, m.fs.stage2, m.fs.stage3, m.fs.mix1, m.fs.mix2 = build_unit_models(m)
 
     add_streams(m)
-    fix_values(m)
+    fix_stream_values(m)
     add_useful_expressions(m)  # adds recovery, purity, and membrane length expressions
 
     return m
 
 
+def add_global_flowsheet_parameters(m):
+    """
+    Adds global parameters to the Pyomo model
+
+    Args:
+        m: Pyomo model
+    """
+    m.Jw = Param(
```

Comment:
This is nice @mollydougher, thanks for adding this. @andrewlee94 should the code specify that is `Param` has `mutable=True`, or is this OK?

Reply:
If a `Param` has units, it is `always `mutable` and will log warnings if this is not set explicitly, so in this case the `Param` should be mutable.

More generally, you should decide if it makes sense for a parameter to change. E.g. acceleration due to gravity is unlikely to change, but a flux parameter like this would change depending on the membrane used and a number of other factors.